packages/builder/src/components/design/settings/controls/ColumnEditor/ColumnDrawer.svelte

```diff
@@ -193,6 +195,9 @@
     display: grid;
     place-items: center;
   }
+  .handle[hide] {
```

Comment:
This would just need updated to `.handle.hide` as well, if you update to use a class rather than a custom attribute.